web/templates/gear_menu.hbs

```diff
@@ -1,7 +1,7 @@
 <ul class="nav" role="navigation">
     <li class="dropdown actual-dropdown-menu" id="gear-menu">
         <a id="settings-dropdown" tabindex="0" role="button" class="dropdown-toggle tippy-zulip-delayed-tooltip" data-target="nada" data-toggle="dropdown" data-tooltip-template-id="gear-menu-tooltip-template">
-            <i class="fa fa-cog settings-dropdown-cog" aria-hidden="true"></i>
+            <i class="fa fa-cog settings-dropdown-cog" aria-label="{{t 'Gear menu'}}"></i>
```

Comment:
This should be called `Menu`, matching the tooltip (`gear-menu-tooltip-template`). 

Or maybe we want something like "Main menu"; @alya thoughts?

I'm also not sure this is the right element to have `aria-label` on; should it be on the `a` tag where the tooltip is declared?

Reply:
Hi Tim, thanks for the review. I agree I should have put the label on the `a` tag. I fixed the pr with your suggestions including changing the lable to `Menu`.

Reply:
"Main menu" sounds good, I guess for both the tooltip and this.

We'll want to update when we do #22802.

Reply:
Ok sounds good. I updated the tooltip and the aria-label. I also updated the `li` tag id to reflect the change to "Main menu".